### DISCUSSION TITLE
What version are folks using?

### DISCUSSION THREAD

**Discussion by wayner9** [OpenSageTV/Community#7] [Docker]
What unRAID OS version are people using now?  I am on 6.11.5, the latest version is 6.12.0rc5.  Anyone on that version yet?  Especially if you are using the DVB plugin for tuner support for stuff like the Hauppauge HD-PVR.

**Comment by jusjoken** [comment]
I too am on 6.11.5 but I do not use the DVB plugin.  My younger self used to install RC versions but not so much any longer.

k

**Reply by wayner9** [reply]
Thanks, now that the Leafs and Raptors are both out I don't have as many worries about missing recordings as there isn't anything critical for me to record that I can't get in other places.
